.changeset/hip-years-clean.md

```diff
@@ -0,0 +1,6 @@
+---
+"@kinobi-so/nodes-from-anchor": patch
+"@kinobi-so/nodes": patch
```

Comment:
I think we can remove this line so the version bump only affects `nodes-from-anchor`.